### PR TITLE
Fix card color and layout in portfolio

### PR DESCRIPTION
**Pull request by stmball**
- Changed card color to match theme
- Changed buttons for better contrast + always positioned at bottom of card
- Slight fixes to syntax to make it more idiomatic to MUI design ideas

### CODE REVIEW

**Comment by stmball** [comment]
Build failing due to the NPM dependency problems you try and get around in the README. Let's try and sort this out?

**Comment by stmball** [comment]
PS: Maybe yarn.lock should be in `.gitignore` as that is what the 12k lines removed are!

File: src/components/portfolio/projects.js [hunk 1]
```diff
@@ -13,16 +20,23 @@ const useStylesProject = makeStyles((theme) => ({
     display: 'flex',
     flexDirection: 'column',
     margin: 10,
-    backgroundColor: '#000000',
-    color: theme.palette.getContrastText(props.background),
-  }),
+    color: theme.palette.getContrastText(theme.palette.primary.main),
```
Comment (szczekulskij) [inline src/components/portfolio/projects.js:23]: Very nice

**Comment by szczekulskij** [comment]
> PS: Maybe yarn.lock should be in `.gitignore` as that is what the 12k lines removed are!

Agreed here. We're not using yarn anyway

**Review by szczekulskij — APPROVED** [review]

**Comment by szczekulskij** [comment]
> Build failing due to the NPM dependency problems you try and get around in the README. Let's try and sort this out?

I'll resolve that post merging

File: src/components/portfolio/projects.js (hunk 1 above)
Comment (stmball) [reply src/components/portfolio/projects.js:23]: i have to be honest im not entirely sure what you're doing with styling on this project... if you're using MUI theme provider i think they encourage to use it online with the sx prop where possible and relate back to the theme via ThemeProvider (in TopLayout.js). also you have styled components as well which i'm a bit confused about!

**Pull request merged**